.github/CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+# https://help.github.com/articles/about-codeowners/
+
+*                         @Roger  @mobilityhouse/qats
```

Comment:
@Roger  I created a new qats team recently can you please give write access to the team?  this should then put everybody as a reviewer [automatically](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/managing-teams-and-people-with-access-to-your-repository#inviting-a-team-or-person)